src/accordion/accordion.ts

```diff
@@ -199,6 +207,14 @@ export class NgbAccordion implements AfterContentChecked {
    */
   isOpen(panelId: string): boolean { return this._states.get(panelId); }
 
+  /**
+   * @internal
```

Comment:
Was pretty sure about the `@internal` but then noticed that it is used on `isOpen`... Need to investigate.

In any case let's try to in-line this method (and yes, you need to return null if you want to remove the attribute altogether).